gateway/ln-gateway/src/lib.rs

```diff
@@ -128,8 +128,13 @@ impl LnGateway {
 
     // Webserver handler for requests to register a federation
     async fn handle_register_federation(&self, payload: RegisterFedPayload) -> Result<()> {
-        let connect: WsFederationConnect =
-            serde_json::from_str(&payload.connect).expect("Invalid federation connect info");
+        let connect: WsFederationConnect = match serde_json::from_str(&payload.connect) {
+            Ok(value) => value,
+            Err(error) => {
+                error!("Invalid member string: {}", error);
+                return Ok(());
```

Comment:
We should probably return the error to the caller since they might not be watching the gateway's log output. 

```rust
.map_err(|_| LnGatewayError::SomeApplicaleVariant))?
```

Reply:
ah yes, that's more useful :+1: